src/Functions/PolygonUtils.h

```diff
@@ -315,7 +315,7 @@ void PointInPolygonWithGrid<CoordinateType>::buildGrid()
     if (has_empty_bound)
         return;
 
-    cells.assign(grid_size * grid_size, {});
+    cells.assign(size_t(grid_size) * grid_size, {});
```

Comment:
Not exploitable because it cannot overflow with our grid size.
We use 16x16 grid, and this arithmetic is Ok up to 256x256 grid.